### ISSUE TITLE
Flux stripes artifacts

### ISSUE THREAD

**Issue by druidui** [kohya-ss/sd-scripts#1948]
Is anyone able to help me understand why my lora is producing stripes lines like this? It trained this on the base flux dev model from huggingface, and am generating with  the same model I have tried disabling "Apply T5 Attention Mask" and retraining, but it has the same effect. I've also tried training different specific block weights as recommended on a similar issue on a reddit thread, but no luck. Thanks 

![Image](https://github.com/user-attachments/assets/d75a77ad-2e96-485b-aa6b-699b8bb8796d)

**Comment by druidui** [comment]
Thinking it could be the fact I'm using sigmoid timestep sampling

**Comment by druidui** [comment]
`{
  "LoRA_type": "Flux1",
  "LyCORIS_preset": "full",
  "adaptive_noise_scale": 0,
  "additional_parameters": "",
  "ae": "E:/stable-diffusion-webui-forge/models/text_encoder/ae.safetensors",
  "apply_t5_attn_mask": false,
  "async_upload": false,
  "block_alphas": "",
  "block_dims": "",
  "block_lr_zero_threshold": "",
  "blocks_to_swap": 0,
  "bucket_no_upscale": true,
  "bucket_reso_steps": 64,
  "bypass_mode": false,
  "cache_latents": true,
  "cache_latents_to_disk": true,
  "caption_dropout_every_n_epochs": 0,
  "caption_dropout_rate": 0,
  "caption_extension": ".txt",
  "clip_g": "",
  "clip_g_dropout_rate": 0,
  "clip_l": "E:/stable-diffusion-webui-forge/models/text_encoder/clip_l.safetensors",
  "clip_skip": 1,
  "color_aug": false,
  "constrain": 0,
  "conv_alpha": 1,
  "conv_block_alphas": "",
  "conv_block_dims": "",
  "conv_dim": 1,
  "cpu_offload_checkpointing": false,
  "dataset_config": "",
  "debiased_estimation_loss": false,
  "decompose_both": false,
  "dim_from_weights": false,
  "discrete_flow_shift": 3,
  "dora_wd": false,
  "double_blocks_to_swap": 0,
  "down_lr_weight": "",
  "dynamo_backend": "no",
  "dynamo_mode": "default",
  "dynamo_use_dynamic": false,
  "dynamo_use_fullgraph": false,
  "enable_all_linear": false,
  "enable_bucket": true,
  "epoch": 1,
  "extra_accelerate_launch_args": "",
  "factor": -1,
  "flip_aug": false,
  "flux1_cache_text_encoder_outputs": true,
  "flux1_cache_text_encoder_outputs_to_disk": true,
  "flux1_checkbox": true,
  "fp8_base": true,
  "fp8_base_unet": false,
  "full_bf16": true,
  "full_fp16": false,
  "gpu_ids": "",
  "gradient_accumulation_steps": 1,
  "gradient_checkpointing": true,
  "guidance_scale": 1,
  "highvram": false,
  "huber_c": 0.1,
  "huber_scale": 1,
  "huber_schedule": "snr",
  "huggingface_path_in_repo": "",
  "huggingface_repo_id": "",
  "huggingface_repo_type": "",
  "huggingface_repo_visibility": "",
  "huggingface_token": "",
  "img_attn_dim": "",
  "img_mlp_dim": "",
  "img_mod_dim": "",
  "in_dims": "",
  "ip_noise_gamma": 0,
  "ip_noise_gamma_random_strength": false,
  "keep_tokens": 0,
  "learning_rate": 0.0005,
  "log_config": false,
  "log_tracker_config": "",
  "log_tracker_name": "",
  "log_with": "",
  "logging_dir": "F:/kohya_ss/lora_wizardcore/log",
  "logit_mean": 0,
  "logit_std": 1,
  "loraplus_lr_ratio": 0,
  "loraplus_text_encoder_lr_ratio": 0,
  "loraplus_unet_lr_ratio": 0,
  "loss_type": "l2",
  "lowvram": false,
  "lr_scheduler": "cosine",
  "lr_scheduler_args": "",
  "lr_scheduler_num_cycles": 1,
  "lr_scheduler_power": 1,
  "lr_scheduler_type": "",
  "lr_warmup": 0,
  "lr_warmup_steps": 0,
  "main_process_port": 0,
  "masked_loss": false,
  "max_bucket_reso": 2048,
  "max_data_loader_n_workers": 0,
  "max_grad_norm": 1,
  "max_resolution": "1024,1024",
  "max_timestep": 1000,
  "max_token_length": 75,
  "max_train_epochs": 15,
  "max_train_steps": 1000,
  "mem_eff_attn": false,
  "mem_eff_save": false,
  "metadata_author": "",
  "metadata_description": "",
  "metadata_license": "",
  "metadata_tags": "",
  "metadata_title": "",
  "mid_lr_weight": "",
  "min_bucket_reso": 256,
  "min_snr_gamma": 7,
  "min_timestep": 0,
  "mixed_precision": "bf16",
  "mode_scale": 1.29,
  "model_list": "custom",
  "model_prediction_type": "raw",
  "module_dropout": 0,
  "multi_gpu": false,
  "multires_noise_discount": 0.1,
  "multires_noise_iterations": 10,
  "network_alpha": 3,
  "network_dim": 6,
  "network_dropout": 0,
  "network_weights": "",
  "noise_offset": 0.03,
  "noise_offset_random_strength": false,
  "noise_offset_type": "Multires",
  "num_cpu_threads_per_process": 1,
  "num_machines": 1,
  "num_processes": 1,
  "optimizer": "AdamW",
  "optimizer_args": "",
  "output_dir": "F:/kohya_ss/lora_wizardcore/model",
  "output_name": "wizardcore",
  "persistent_data_loader_workers": false,
  "pos_emb_random_crop_rate": 0,
  "pretrained_model_name_or_path": "E:/stable-diffusion-webui-forge/models/Stable-diffusion/flux1-dev.safetensors",
  "prior_loss_weight": 1,
  "random_crop": false,
  "rank_dropout": 0,
  "rank_dropout_scale": false,
  "reg_data_dir": "",
  "rescaled": false,
  "resume": "",
  "resume_from_huggingface": "",
  "sample_every_n_epochs": 1,
  "sample_every_n_steps": 0,
  "sample_prompts": "wizardcore, a cat wearing a wizard hat, night sky, stars, castle in the distance on a hill --w 832 --h 1216 --s 20 --l 4 --d 42",
  "sample_sampler": "euler",
  "save_clip": false,
  "save_every_n_epochs": 1,
  "save_every_n_steps": 0,
  "save_last_n_epochs": 0,
  "save_last_n_epochs_state": 0,
  "save_last_n_steps": 0,
  "save_last_n_steps_state": 0,
  "save_model_as": "safetensors",
  "save_precision": "bf16",
  "save_state": false,
  "save_state_on_train_end": false,
  "save_state_to_huggingface": false,
  "save_t5xxl": false,
  "scale_v_pred_loss_like_noise_pred": false,
  "scale_weight_norms": 0,
  "sd3_cache_text_encoder_outputs": false,
  "sd3_cache_text_encoder_outputs_to_disk": false,
  "sd3_checkbox": false,
  "sd3_clip_l": "",
  "sd3_clip_l_dropout_rate": 0,
  "sd3_disable_mmap_load_safetensors": false,
  "sd3_enable_scaled_pos_embed": false,
  "sd3_fused_backward_pass": false,
  "sd3_t5_dropout_rate": 0,
  "sd3_t5xxl": "",
  "sd3_text_encoder_batch_size": 1,
  "sdxl": false,
  "sdxl_cache_text_encoder_outputs": true,
  "sdxl_no_half_vae": true,
  "seed": 650254876,
  "shuffle_caption": false,
  "single_blocks_to_swap": 0,
  "single_dim": "",
  "single_mod_dim": "",
  "skip_cache_check": false,
  "split_mode": false,
  "split_qkv": false,
  "stop_text_encoder_training": 0,
  "t5xxl": "E:/stable-diffusion-webui-forge/models/text_encoder/t5xxl_fp16.safetensors",
  "t5xxl_device": "",
  "t5xxl_dtype": "bf16",
  "t5xxl_lr": 0,
  "t5xxl_max_token_length": 512,
  "text_encoder_lr": 0,
  "timestep_sampling": "sigmoid",
  "train_batch_size": 1,
  "train_blocks": "all",
  "train_data_dir": "F:/kohya_ss/lora_wizardcore/img",
  "train_double_block_indices": "all",
  "train_norm": false,
  "train_on_input": true,
  "train_single_block_indices": "all",
  "train_t5xxl": false,
  "training_comment": "",
  "txt_attn_dim": "",
  "txt_mlp_dim": "",
  "txt_mod_dim": "",
  "unet_lr": 0.0003,
  "unit": 1,
  "up_lr_weight": "",
  "use_cp": false,
  "use_scalar": false,
  "use_tucker": false,
  "v2": false,
  "v_parameterization": false,
  "v_pred_like_loss": 0,
  "vae": "",
  "vae_batch_size": 0,
  "wandb_api_key": "",
  "wandb_run_name": "",
  "weighted_captions": false,
  "weighting_scheme": "logit_normal",
  "xformers": "sdpa"
}`
